Dockerfile.api

```diff
@@ -4,9 +4,12 @@ RUN mkdir -p /data/gamutrf
 COPY requirements.txt /root/requirements.txt
 RUN pip3 install -r /root/requirements.txt
 RUN git clone https://github.com/Nuand/bladeRF.git -b 2021.10
+RUN git clone https://github.com/anarkiwi/lime-tools -b samples
 WORKDIR /root/bladeRF/host/build
 RUN cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local -DINSTALL_UDEV_RULES=ON -DENABLE_BACKEND_LIBUSB=TRUE .. && make -j "$(nproc)" && make install && ldconfig -v
 COPY . /root/gamutrf
+WORKDIR /root/lime-tools/build
+RUN cmake .. && make install
```

Comment:
this still needs a `make` step, no?

Reply:
implicitly executes make all by make install